SQL_agg_functions/AdventureWorks-for-Postgres/sql-scripts/select.sql

```diff
@@ -0,0 +1,72 @@
+--1. Identify Department Groupings
+SELECT COUNT(DISTINCT humanresources.department.name) AS number_of_departments, 
+        humanresources.department.groupname
+FROM humanresources.department
+GROUP BY humanresources.department.groupname
+ORDER BY number_of_departments DESC;
+
+--2. Determine Maximum Salary for Each Employee
+SELECT humanresources.employee.businessentityid, 
+    humanresources.employee.loginid, 
+    humanresources.employee.jobtitle,
+    MAX(humanresources.employeepayhistory.rate) AS maximum_salary
+FROM humanresources.employee
+JOIN humanresources.employeepayhistory 
+    ON humanresources.employee.businessentityid = humanresources.employeepayhistory.businessentityid
+GROUP BY humanresources.employee.businessentityid, 
+    humanresources.employee.loginid, 
+    humanresources.employee.jobtitle
+ORDER BY maximum_salary DESC, 
+    humanresources.employee.businessentityid;
+
+--3. Minimum Product Price by Subcategory
+SELECT production.productsubcategory.productsubcategoryid,
```

Comment:
It is a good practice to use aliases in tour queries, instead of explicitly mentioning each time schema and table name you can replace **production.productsubcategory** with some alias like `product_sub_category` or replace **production.product** with **product**, and so on